### PR TITLE
Hide search modifiers when symbol search enabled

### PR DESCRIPTION
**Pull request by anbarasiu**
Associated Issue: #2592 

### Summary of Changes

* Modified the disabled property to hide search modifiers, when symbol search enabled, else to show them.

Please let me know if this looks okay.

### CODE REVIEW

File: src/components/Editor/SearchBar.js [hunk 1]
```diff
@@ -624,7 +624,7 @@ class SearchBar extends Component {
         {
           className: classnames(className, {
             active: !symbolSearchOn && modifiers && modifiers.get(modVal),
-            disabled: symbolSearchOn
+            hide: symbolSearchOn
```
Comment (jasonLaster) [inline src/components/Editor/SearchBar.js]: this is a valid approach, but we often just choose not to render a component too :)

i.e.

```js
if (! symbolSearchOn) {
  return null;
}
```

Comment (anbarasiu) [inline src/components/Editor/SearchBar.js]: That seems a better way of doing it. I've modified it to not render the component. Please take a look and let me know :)

**Comment by anbarasiu** [comment]
Great, thanks!

**Comment by jasonLaster** [comment]
Hey @anbarasiu, you should join our slack if you haven't already: https://devtools-html.slack.com

**Comment by anbarasiu** [comment]
Thanks! I've joined :)

**Pull request merged**